homeassistant/components/rfxtrx/__init__.py

```diff
@@ -212,6 +215,9 @@ async def async_unload_entry(hass, entry: config_entries.ConfigEntry):
 
     hass.services.async_remove(DOMAIN, SERVICE_SEND)
 
+    for cleanup_callback in hass.data[DATA_CLEANUP_CALLBACKS]:
```

Comment:
You also need to clean up the data from `hass.data`

Reply:
Oops, that was meant to be `hass.data[DOMAIN][DATA_CLEANUP_CALLBACKS]`